### DISCUSSION TITLE
Siemens logo

### DISCUSSION THREAD

**Discussion by PaulRohlik** [frangoteam/FUXA#730] [Q&A] [answered]
Hello.
Please help again. I want to connect siemens logo via S7. But I can't get the data from the plc.
Node red is also running on the same guest PC and everything works there.
Can someone please advise?
![pjoblem](https://user-images.githubusercontent.com/48689104/224770742-d7461e87-9904-4deb-bf2f-e9e72800123b.png)

Very thans.

**Comment by ProfDrOetker** [comment]
You might try to use simple tag addresses like `M0.1` or `Q0.1` If that is working you can dive deeper to the `DB.X` or similar addresses. I so not know how these are handled inside a Logo.

**Reply by PaulRohlik** [reply]
Thank you. Addresses Q0.1 ... work.
But what do the addresses for:
DB1, Xxxx
DB1, Wxxx
DB1,INTxxx

**Reply by PaulRohlik** [reply]
I've got it :)
![s7](https://user-images.githubusercontent.com/48689104/225530543-20811bf3-348c-4237-a6cf-62ffb1ae0ac9.png)

**Reply by bastionji** [reply, marked as answer]
Hi, you can use modbus connection to read and write values on Siemens Logo!.

![image](https://user-images.githubusercontent.com/128835833/227593207-7784f3f3-0994-4458-9db7-a53fce7880c3.png)

but remember to activate the communication before....

![image](https://user-images.githubusercontent.com/128835833/227593839-a875ade6-d2c4-4b3d-ac0c-2b97fe538b86.png)

**Comment by jbryan189** [comment]
Just went thru this exercise and found the Siemens LOGO uses Port 510 in ModbusTCP into my application of FUXA.

**Comment by wagod835512** [comment]
Hi,

Variable mapping for S7 communication is specified in link below. You need to enable and configure S7 communication as server in LOGO project.

https://support.industry.siemens.com/cs/mdm/100782807?c=85315142923&lc=el-GR

**Comment by PaulRohlik** [comment]
Hello,
The problem was in the address format. It works perfectly now :)
